stream_alert/rule_processor/parsers.py

```diff
@@ -255,13 +275,19 @@ def _parse_records(self, schema, json_payload):
                 return False
             for match in matches:
                 record = match.value
+                embedded_json = self.options.get('embedded_json')
+                if embedded_json:
+                    try:
+                        record = json.loads(match.value)
+                    except ValueError:
+                        LOGGER.debug('Embedded json is invalid')
```

Comment:
Should this be a warning? If I understand correctly, this logic parses embedded json (if specified) for each matched log schema. If that fails, then the config is probably wrong? Or will this happen more frequently?

Reply:
It's probably safe to have it as warning or error. just updated!